code/modules/mob/living/simple_animal/hostile/terror_spiders/chem.dm

```diff
@@ -0,0 +1,43 @@
+#define SOLID 1
+#define LIQUID 2
+#define GAS 3
+#define FOOD_METABOLISM 0.4
+#define REM REAGENTS_EFFECT_MULTIPLIER
+
+
+// Terror Spider, Black, Deadly Venom
+
+/datum/reagent/terror_black_toxin
+	name = "Black Widow venom"
+	id = "terror_black_toxin"
+	description = "An incredibly toxic venom injected by the Black Widow spider."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.1
+
+/datum/reagent/terror_black_toxin/on_mob_life(var/mob/living/M)
```

Comment:
Don't use implied vars in procs

`/datum/reagent/terror_black_toxin/on_mob_life(mob/living/M)`

Is what this should be